src/test/fuzz/txorphan.cpp

```diff
@@ -159,6 +204,8 @@ FUZZ_TARGET(txorphan, .init = initialize_orphanage)
                 });
 
         }
+        orphanage.SanityCheck();
```

Comment:
nit: this could be moved completely outside the loops for the same checking power

Reply:
Isn't it already out of the loop?

Reply:
Oh nvm, there are 2 loops, yes

Reply:
done